.github/workflows/wheel_win_x86_64.yml

```diff
@@ -206,22 +206,22 @@ jobs:
       - uses: actions/upload-artifact@v2
         if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
         with:
-          name: ${{ runner.os }}-wheels.zip
-          path: Z:\dist\*.whl
+          name: ${{ runner.os }}-wheels-${{ matrix.arch }}.zip
+          path: ./wheelhouse/*.whl
 
   upload-pypi:
     needs: win-wheels
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

Comment:
I guess the main issue was the wheels built already with the release version, and uploaded as part of the test workflow, due to the above line being commented out. I think this is fine to uncomment the lines here and improve our process for testing these in future to avoid the duplicate wheel uploads causing the failure.

Reply:
Or maybe I am misunderstanding the given issue -- is it a path issue to the zip file then?

Reply:
We had a path issue, this comment is for testing. I will let you know when it is in its final form.

Reply:
Maybe best to keep in draft form then for now.